.github/workflows/5_e2e_tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Integration tests 
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      extension_version:
+        description: 'VSCode version'
+        required: true
+      branch:
+        description: 'Branch to run run this workflow on'
+        required: true
+
+jobs:
+  tests:
+    name: Integration tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Checkout branch
+        run: |
+          git checkout ${{ github.event.inputs.branch }}
+      - name: Install Dependencies
+        run: npm install
+      - name: Compile
+        run: npm run compile
+      - name: Headless e2e tests
+        uses: GabrielBB/xvfb-action@v1.0
+        with:
+          run: npm run test
```

Comment:
How does this look like when we have https://github.com/prisma/language-tools/issues/440?
We won't need to publish LSP in step 3 then but somewhere here, right?

Reply:
This is resolved I think?

Reply:
Yes, #440 only affects the workflow run on a PR, which did not get changed in the current PR 👍